website/content/docs/install/ports.mdx

```diff
@@ -21,13 +21,14 @@ Before running Consul, you should ensure the following bind ports are accessible
 | HTTP: The HTTP API (TCP Only)                                                                                 | 8500              |
 | HTTPS: The HTTPs API                                                                                          | disabled (8501)\* |
 | gRPC: The gRPC API                                                                                            | disabled (8502)\* |
+| gRPC: The gRPCs API                                                                                           | 8403              |
 | LAN Serf: The Serf LAN port (TCP and UDP)                                                                     | 8301              |
 | Wan Serf: The Serf WAN port (TCP and UDP)                                                                     | 8302              |
 | server: Server RPC address (TCP Only)                                                                         | 8300              |
 | Sidecar Proxy Min: Inclusive min port number to use for automatically assigned sidecar service registrations. | 21000             |
 | Sidecar Proxy Max: Inclusive max port number to use for automatically assigned sidecar service registrations. | 21255             |
 
-\*For `HTTPS` and `gRPC` the ports specified in the table
+\*For `HTTPS` and `gRPC(s)` the ports specified in the table
```

Comment:
```suggestion
\*For `HTTPS` and `gRPC(s)` the ports specified in the table
```
What is there a `(s)` after `gRPC`? Do we mean that the ports for HTTP, gRPC, and the gRPC API are recommendations? Or do we mean `gRPC` and `gRPCs`?

Reply:
The trailing s should indicate that you can (or indeed should) use TLS for your grpc connections. We can omit it to clarify the documentation